src/docker/DockerodeApiClient/DockerodeApiClient.ts

```diff
@@ -255,7 +256,12 @@ export class DockerodeApiClient extends ContextChangeCancelClient implements Doc
     }
 
     public async getImages(context: IActionContext, token?: CancellationToken): Promise<DockerImage[]> {
-        const images = await this.callWithErrorHandling(context, async () => this.dockerodeClient.listImages({ filters: { "dangling": ["false"] } }), token);
+        const filters = {};

```

Comment:
A semantic question, do we intend the toggle to control whether dangling images are included in all requests, or only when the UI actually *displays* a list of images?  If the latter, then perhaps `getImages()` should accept a `getDanglingImages` argument and the tree view manage getting the setting from global state.

Reply:
Great point. My vote would be for the setting to only apply to the UI (the second alternative)

Reply:
I vote to only apply to the UI.

Reply:
We are in agreement then. 

@jnsjunior please have the `getImages` function take an optional `includeDangling` parameter (defaulting to false), then have the [images tree item (plural)](https://github.com/microsoft/vscode-docker/blob/main/src/tree/images/ImagesTreeItem.ts#L60) check the `showDanglingImages` state parameter and call `getImages` accordingly. 

Thank you for working with us through the review!!

Reply:
@karolz-ms and @philliphoff thank you for support me for my first contribution for the vscode-docker :smiley: